src/main/java/com/hrznstudio/titanium/api/redstone/IRedstoneState.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This file is part of Titanium
+ * Copyright (C) 2020, Horizon Studio <contact@hrznstudio.com>.
+ *
+ * This code is licensed under GNU Lesser General Public License v3.0, the full license text can be found in LICENSE.txt
+ */
+
+package com.hrznstudio.titanium.api.redstone;
+
+public interface IRedstoneState {
+
+    boolean isReceivingRedstone();
```

Comment:
I assume this is basically isPowered but more wordy?

Reply:
Yes